src/styles/layouts/comm-layout.scss

```diff
@@ -0,0 +1,58 @@
+.comm-hero {
+  margin-top: $base-height * 8;
+}
+
+.comm-header {
+  margin-bottom: $base-height * 4;
+  font-size: 28px;
+  line-height: $base-height * 5;
+  color: $white;
+  text-align: center;
+
+  @include media-breakpoint-up(lg) {
+    text-align: left;
+  }
+}
+
+.comm-container {
+  padding: 0;
+  text-align: center;
+
+  @include media-breakpoint-up(lg) {
+    text-align: left;
+  }
+}
+
+.comm-text {
+  margin-bottom: $base-height * 6;
+  color: $white;
+  text-align: center;
+
+  @include media-breakpoint-up(lg) {
+    text-align: left;
+  }
+}
+
+.btn-forum {
+  padding: 0 $base-height * 7;
+  background-color: $aurora-blue;
+  color: $white;
+  font-family: 'Poppins';
```

Comment:
Should we also have SCSS variables for fonts?

Reply:
Yeah, that's a damn good idea considering how often they change from one element to the next. Thanks for the suggestion!